submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/filter/PaginationFilter.kt

```diff
@@ -0,0 +1,6 @@
+package ac.uk.ebi.biostd.persistence.filter
+
+open class PaginationFilter(val limit: Int = 15, private val offset: Int = 0) {
```

Comment:
I think we need to separate filter from paging for me both concepts are different and should not be coupled. `SubmissionDraftClient#getAllSubmissionDrafts` should receive in my opinion and offset and a limit as a parameter not a map that includes this two and user need to remember paging parameter names

Reply:
Done